hack/update-codegen-crds.sh

```diff
@@ -46,7 +49,7 @@ done
 #     crd \
 #     rbac:roleName=manager-role \
 #     webhook \
-#     paths="./test/e2e/reconciler/cluster/..." \
-#     output:crd:artifacts:config=test/e2e/reconciler/cluster/
+#     paths="${REPO_ROOT}/test/e2e/reconciler/cluster/..." \
+#     output:crd:artifacts:config="${REPO_ROOT}"/test/e2e/reconciler/cluster/
 
-# go run ./cmd/apigen --input-dir ./config/crds --output-dir  ./config/root-phase0
+"${REPO_ROOT}"/hack/tools/apigen --input-dir "${REPO_ROOT}"/config/crds --output-dir "${REPO_ROOT}"/config/exports
```

Comment:
What happened to the version number in that binary name?
Don't you mean to use `$API_GEN` ?

Reply:
'apigen' is logical linked to the fully qualified apigen binary that includes the version number in the name.

Reply:
I changed run line to use ${API_GEN} which includes path and version number in name